sqlmesh/core/selector.py

```diff
@@ -74,23 +76,24 @@ def select_models(
 
         dag: DAG[str] = DAG()
         models: UniqueKeyDict[str, Model] = UniqueKeyDict("models")
-        all_model_names = set(self._models) | set(env_models)
-        for name in all_model_names:
+
+        all_model_fqns = set(self._models) | set(env_models)
```

Comment:
Since we no longer normalize model names shouldn't we go back to using names here instead of fqns?

Reply:
A few lines up we get the expanded model selections and that returns fqn and we reference that below here. So having everything be fqn here makes all these checks line up. Is there a downside I am not considering?

Reply:
ok, makes sense, thank you